### PR TITLE
[#16] [UI] Implement carousel card UI

### LINKED ISSUE #16: [UI] As a logged in user, I can see the Survey detail at the bottom.
## Why
- Show the detail of the surveys on the home screen, it's a list item detail, which consists of:
  - Survey title
  - Survey description
  - Float button to access the detail.
## Acceptance Criteria

image

### PR DESCRIPTION
**Pull request by sleepylee**
Closes #16
## What happened 👀

Implement the card UI (single card) - prepare for the Carousel (PageView) list implementation next.
 
## Insight 📝
- Basic UI learning & implementation.
- Mocked data will be removed later.

### CODE REVIEW

**Review by markgravity — APPROVED** [review]
LGTM 🤖

File: lib/pages/home/survey_carousel/survey_carousel_card.dart [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+// ignore: must_be_immutable
+class SurveyCarouselCard extends StatelessWidget {
+  SurveyCarouselCard({this.surveyUiModel});
+
+  final SurveyUiModel surveyUiModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
```
Comment (manh-t) [inline lib/pages/home/survey_carousel/survey_carousel_card.dart:13]: redundant `Center` widget imo

File: lib/pages/home/survey_carousel/survey_carousel_card.dart [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      child: Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
```
Comment (manh-t) [inline lib/pages/home/survey_carousel/survey_carousel_card.dart:16]: by default `Container` widget will take as much as possible size, so we don't need define max size here 🙆‍♂️

File: lib/pages/home/survey_carousel/survey_carousel_card.dart [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        decoration: BoxDecoration(
+            image: DecorationImage(
+          image: NetworkImage(surveyUiModel.imageUrl),
+          fit: BoxFit.cover,
+          colorFilter: ColorFilter.mode(
+              Colors.black.withOpacity(0.5), BlendMode.overlay),
+        )),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.max,
```
Comment (manh-t) [inline lib/pages/home/survey_carousel/survey_carousel_card.dart:26]: default axisSize is `max` already

**Review by manh-t — COMMENTED** [review]
this branch needs rebasing 🙏

**Comment by sleepylee** [comment]
all updated, thanks @manh-t for some new useful info.
The app worked normally:

**Review by manh-t — APPROVED** [review]
LGTM 🚀

**Pull request merged**